docs/firmwarePost.md

```diff
@@ -0,0 +1,30 @@
+# POST /firmware
+
+## Upload a firmware file
+
+Upload a firmware file that could be used in scopes
+Use RFC1867
+
+Example request:
+```text
+curl -kv -F "firmware=@/etc/hosts" 'https://192.168.122.76/tancredi/api/v1/firmware' -H 'Accept: application/json, text/plain, */*' --compressed -H 'User: admin' -H 'Secretkey: 71531d20e69d0e9d83a666e9b4fd5fae11dcdbb1'
+```
```

Comment:
I'd like to see here the HTTP request (not a curl command), with the headers and a sample payload.

Maybe `curl -i` can provide a good starting point